### ISSUE TITLE
Device summary widget shows 2 down ports when no ports are down

### ISSUE THREAD

**Issue by clearmoon247** [librenms/librenms#2568]
When using either the horizontal or vertical device summary widget, it shows the ports down as 2, but when I click the link (/ports/format=list_detail/state=down/) to see the down ports, there are no entries. 

![device_summary_2](https://cloud.githubusercontent.com/assets/3594299/11629843/77337b06-9cc8-11e5-8d25-20ead358df01.JPG)
![port-list-0](https://cloud.githubusercontent.com/assets/3594299/11629844/77371ec8-9cc8-11e5-8e78-d1a5d4168158.JPG)

**Comment by laf** [comment]
The ports page can be a bit misleading sometimes, we don't show disabled ports by default in the port listing so I expect if you click the disabled tickbox and search again you will then see the two missing ports.

Their is a reason why it's like which has been discussed in a previous issue.

**Comment by clearmoon247** [comment]
When I show disabled ports, my list shows ~2,681 devices when I click disabled and search.

**Comment by laf** [comment]
Are you now searching just disabled, if so that's not what I was referring to, it was searching down AND disabled. See if that shows up 2?

**Comment by clearmoon247** [comment]
I was able to confirm that when i search for down and disabled that is what my results are.

![down_disabled](https://cloud.githubusercontent.com/assets/3594299/11692818/848382bc-9e6f-11e5-92e0-b3a02db6c3f0.JPG)
![down_disabled_results](https://cloud.githubusercontent.com/assets/3594299/11692817/84832c9a-9e6f-11e5-90c2-b8c9017b4c9f.JPG)

**Comment by laf** [comment]
Is this an admin user or non-admin?

Can you run this mysql query:

`SELECT COUNT(*) FROM ports AS I, devices AS D WHERE I.`deleted` = '0' AND D.`device_id` = I.`device_id` AND I.`ignore` = '0' AND D.`ignore` = '0' AND I.`ifOperStatus` = 'down' AND I.`ifAdminStatus` = 'up'`

**Comment by clearmoon247** [comment]
That was for an admin. When I ran that command in MySql, it gave the count of 2. I ran that command again, but with * instead of COUNT(*), I was able to find the two ports that it was counting.

Both ports in Port Settings were in an Up/Down state, and the settings were Disabled was checked, but Ignore was not.

Once I checked ignore, the number of down ports is now 0
![port-list-fixed](https://cloud.githubusercontent.com/assets/3594299/11744889/fb53564c-9fe1-11e5-9cba-123bf2db02a4.JPG)
